response-items/src/ellipse-scale/ResponseItem.stats.js

```diff
@@ -43,6 +43,8 @@ const getPlotlyProps = (iaa, minMax) => {
   data = { ...newData }
   const maxY = Math.max(...data.y);
   const centroidValue = iaa.centroid;
+  const mean_of_midpoints = iaa.mean_of_midpoints
```

Comment:
This is gonna sound petty, but can we camelCase everything in here?

1. It's the more common JS convention, as opposed to python's snake_case
2. More importantly it's consistent with the immediately surrounding codebase e.g. `centroidValue`, `dataTrace`, `centroidTrace` etc...